### PR TITLE
Remove empty strings from locale list

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by evgeni** [comment]
I think the first list isn't required anymore, but doesn't hurt either.

Otherwise clever abuse of '' being False-y, and you know what I think about clever code ;)

`select("ne", "")` might be more obvious ;) (untested)

Why does your env have those vars set to empty anyways?

**Comment by ekohl** [comment]
You know I'm a fan of context and should complain about the lack of it in this PR. The context is https://community.theforeman.org/t/foreman-nightly-deb-pipeline-580-failed/22454.

**Comment by evgeni** [comment]
aha, yeah, that makes sense!

**Comment by ehelms** [comment]
I don't know what you want me to do here.

File: roles/foreman_installer/tasks/locales.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 - name: 'Ensure ENV locales are available'
   locale_gen:
     name: "{{ item }}"
-  loop: "{{ query('env', 'LANG', 'LC_ADDRESS', 'LC_ALL', 'LC_COLLATE', 'LC_CTYPE', 'LC_IDENTIFICATION', 'LC_MEASUREMENT', 'LC_MESSAGES', 'LC_MONETARY', 'LC_NAME', 'LC_NUMERIC', 'LC_PAPER', 'LC_TELEPHONE', 'LC_TIME') | list | unique }}"
+  loop: "{{ query('env', 'LANG', 'LC_ADDRESS', 'LC_ALL', 'LC_COLLATE', 'LC_CTYPE', 'LC_IDENTIFICATION', 'LC_MEASUREMENT', 'LC_MESSAGES', 'LC_MONETARY', 'LC_NAME', 'LC_NUMERIC', 'LC_PAPER', 'LC_TELEPHONE', 'LC_TIME') | list | unique | select() | list }}"
```
Comment (evgeni) [inline roles/foreman_installer/tasks/locales.yml:14]:
```suggestion
  loop: "{{ query('env', 'LANG', 'LC_ADDRESS', 'LC_ALL', 'LC_COLLATE', 'LC_CTYPE', 'LC_IDENTIFICATION', 'LC_MEASUREMENT', 'LC_MESSAGES', 'LC_MONETARY', 'LC_NAME', 'LC_NUMERIC', 'LC_PAPER', 'LC_TELEPHONE', 'LC_TIME') | unique | select("ne", "") | list }}"
```

Comment (evgeni) [reply roles/foreman_installer/tasks/locales.yml:14]: @ehelms that ;)

Comment (ehelms) [reply roles/foreman_installer/tasks/locales.yml]: That lacks a certain.... panache of cleverness. But Ok I did it :) Thanks for the snippet

**Review by evgeni — APPROVED** [review]
you can still drop the first `list` as in my suggestion, but I don't care too much ;)

**Pull request merged**